plugins/nextcloud-rule-exclusions-before.conf

```
@@ -1062,6 +1062,22 @@ SecRule REQUEST_FILENAME "@rx /apps/photos/api/v[0-9]/config/config/(?:croppedLa
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# Photos: Albums, Shared Albums, Places
+# Allow the data type 'text/plain'
+# Since the content is actually XML, we switch on the XML parser
+SecRule REQUEST_FILENAME "@rx ^/remote\.php/dav/photos/(?:[a-zA-Z0-9\-\_\. \@]|\%40)*/(?:(?:shared)?albums|places)/" \
```

This should work. Please test.
```suggestion
SecRule REQUEST_FILENAME "@rx ^/remote\.php/dav/photos/[^/]+/(?:(?:shared)?albums|places)$" \
```

---

I agree with @theseion to allow all characters within the username field even if Nextcloud only allows a certain set of characters. I don't think there will be much gain (if any) in limiting it, plus it looks a little cleaner that way. Also, the suggested regex although stricter does have FPs, this one ``/remote\.php/dav/photos/[^/]+/(?:albums|sharedalbums|places)(?:/[^/]+)?/$
`` from my testing it works well and accommodates users that may have installed nextcloud under ``/nextcloud/`` or ``/my-nextcloud/`` for example.